### DISCUSSION TITLE
posts/%E5%89%8D%E5%A5%B3%E5%8F%8B%E6%98%AF%E4%BD%A0%E5%92%8C%E5%A5%B9%E7%A4%BE%E4%BC%9A%E5%85%B3%E7%B3%BB%E7%9A%84%E6%80%BB%E5%92%8C%E8%AE%B0%E5%B8%95%E9%9C%B2%E8%8F%B2%E9%87%8C%E4%BC%BD%E5%AD%90%E7%BA%BF/

### DISCUSSION THREAD

**Discussion by giscus** [Flying-dragon-boxing/BlogDiscussions#5] [Announcements]
# posts/%E5%89%8D%E5%A5%B3%E5%8F%8B%E6%98%AF%E4%BD%A0%E5%92%8C%E5%A5%B9%E7%A4%BE%E4%BC%9A%E5%85%B3%E7%B3%BB%E7%9A%84%E6%80%BB%E5%92%8C%E8%AE%B0%E5%B8%95%E9%9C%B2%E8%8F%B2%E9%87%8C%E4%BC%BD%E5%AD%90%E7%BA%BF/

丸户老师的货劲就是大。 为什么酒是陈的香，酒好是一方面，陈年让口感更丰富也只是另一个方面罢了。或许，“陈年”这个词里蕴含的社会价值，比如保存老

https://blog.rhx0820.xyz/posts/%E5%89%8D%E5%A5%B3%E5%8F%8B%E6%98%AF%E4%BD%A0%E5%92%8C%E5%A5%B9%E7%A4%BE%E4%BC%9A%E5%85%B3%E7%B3%BB%E7%9A%84%E6%80%BB%E5%92%8C%E8%AE%B0%E5%B8%95%E9%9C%B2%E8%8F%B2%E9%87%8C%E4%BC%BD%E5%AD%90%E7%BA%BF/

**Comment by Flying-dragon-boxing** [comment]
最近太忙了，完全没时间更
